.github/workflows/build-linux.yml

```
@@ -0,0 +1,45 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.22.7"
+
+      - name: Set env vars
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2193:warning:1:27: The arguments to this comparison can never be equal. Make sure your syntax is correct

(shellcheck)